package.json

```diff
@@ -2891,7 +2891,6 @@
         "tar": "^6.1.11",
         "tree-kill": "^1.2.2",
         "vscode-languageclient": "^8.0.2",
-        "vscode-nls": "^5.0.0",
```

Comment:
I can't put the comment in the right location, but we need to update `@types/vscode` to 1.75.0, and also update the VSCode engine dependency to match.

Reply:
I see. Good catch!